ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/validator/preexecution/LogAppenderPreExecutingCommitStrategySpec.scala

```diff
@@ -45,11 +45,11 @@ class LogAppenderPreExecutingCommitStrategySpec
         actual =>
           verify(mockStateKeySerializationStrategy, times(preExecutionResult.stateUpdates.size))
             .serializeStateKey(any[DamlStateKey])
-          actual.successWriteSet should have size (preExecutionResult.stateUpdates.size + 1L)
-          actual.successWriteSet.toMap.keys should contain(expectedLogEntryKey)
+          actual.successWriteSet.state should have size preExecutionResult.stateUpdates.size.toLong
+          actual.successWriteSet.logEntry._1 should be(expectedLogEntryKey)
```

Comment:
If you change the `RawKeyValuePairsWithLogEntry` type as suggested you won't have to use `_1` here.